frontend/src/pages/ForgotPassword/ForgotPassword.jsx

```diff
@@ -79,22 +142,46 @@ export function ForgotPassword() {
             ) : (
               <React.Fragment>
                 <h1 className={style["card-heading"]}>Trouble Logging In?</h1>
-                <form onSubmit={submitForgotPassword}>
+                <form
+                  onSubmit={() => {
+                    handleSubmit();
+                    submitForgotPassword();
+                  }}
+                >
                   <div className={style["inside-card"]}>
-                    <div className={style["forgot-password-input"]}>
-                      <input
-                        autocomplete="off"
-                        id="username"
-                        type="email"
-                        required="required"
-                        name="email"
-                        placeholder="Email"
-                        onChange={handleChange}
-                        className={style["input-forgot-password"]}
-                        onInvalid={(e)=>{ e.target.setCustomValidity("Enter a valid Email Id") }}
-                        onInput={(e)=>{ e.target.value?.match(new RegExp(/^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/)) && e.target.setCustomValidity("") }}
-                      />
-                      <i className="fas fa-at"></i>
+                    <div className={`form-group ${style["form-group"]}`}>
+                      <div className={style["forgot-password-input"]}>
+                        <input
+                          autocomplete="off"
+                          id="username"
+                          type="email"
+                          name="email"
+                          placeholder="Email"
+                          onChange={handleChange}
+                          required="required"
+                          className={style["input-forgot-password"]}
+                          onInvalid={(e) => {
+                            e.target.setCustomValidity(
```

Comment:
as we are using `joi-validator` so do we need this code snippet from here to [here](https://github.com/HITK-TECH-Community/Community-Website/pull/540/files#diff-9f3cf1c20eff239632332e6daf33e42638fa5cb5bca0ddff9b203afc43c3ef37R175)